apps/voting/app/src/App.js

```diff
@@ -16,14 +16,11 @@ import VotePanelContent from './components/VotePanelContent'
 import NewVotePanelContent from './components/NewVotePanelContent'
 import { networkContextType } from './utils/provideNetwork'
 import { settingsContextType } from './utils/provideSettings'
-import { makeEtherscanBaseUrl } from './utils'
 import { hasLoadedVoteSettings } from './vote-settings'
 import { VOTE_YEA } from './vote-types'
-import {
-  EMPTY_CALLSCRIPT,
-  isVoteOpen,
-  voteTypeFromContractEnum,
-} from './vote-utils'
+import { makeEtherscanBaseUrl } from './utils'
```

Comment:
We could start using `blockExplorerUrl()` (it’s available in this version of Aragon UI): https://github.com/aragon/aragon-ui/blob/master/src/utils/web3.js#L69

Reply:
Let's wait for the next version and then upgrade them all at the same time :).